docker/config/local_dev.env

```diff
@@ -17,6 +17,8 @@ SECRET_KEY=DontusethisinproductionbutitneedsbelongforCI1234567890
 
 DATABASE_URL=postgresql://postgres:postgres@db/tecken
 
+REDIS_URL=redis://redis-cache:6379/0
```

Comment:
I forgot this got into this PR. I can probably extract it out into a separate PR, but what's going on here is that the local dev environment redis url was being used as a default value in the configuration and that's not how we want to set things up. The defaults should be good for a server environment.

Given that, I pulled this value out into this `local_dev.env` file and removed the default. Not having it set is a server configuration error and should show up as such.